src/main/java/teammates/common/util/Templates.java

```diff
@@ -411,21 +411,30 @@ public static String populateTemplate(String template, String... keyValuePairs)
-            public static final String RANK_MAX_OPTIONS_TO_BE_RANKED = "${maxOptionsToBeRanked}";
             public static final String RANK_ARE_DUPLICATES_ALLOWED_VALUE = "${areDuplicatesAllowedValue}";
             public static final String RANK_ARE_DUPLICATES_ALLOWED_CHECKED = "${areDuplicatesAllowedChecked}";
-            public static final String RANK_DISPLAY_MIN_OPTIONS_HINT = "${displayMinOptionsHint}";
-            public static final String RANK_IS_MIN_OPTIONS_TO_BE_RANKED_ENABLED = "${isMinOptionsToBeRankedEnabled}";
-            public static final String RANK_DISPLAY_MAX_OPTIONS_HINT = "${displayMaxOptionsHint}";
-            public static final String RANK_IS_MAX_OPTIONS_TO_BE_RANKED_ENABLED = "${isMaxOptionsToBeRankedEnabled}";
             public static final String RANK_SUBMISSION_FORM_OPTION_FRAGMENTS = "${rankSubmissionFormOptionFragments}";
             public static final String RANK_OPTION_RECIPIENT_DISPLAY_NAME = "${optionRecipientDisplayName}";
+            public static final String RANK_MIN_OPTIONS_TO_BE_RANKED = "${minOptionsToBeRanked}";
+            public static final String RANK_MAX_OPTIONS_TO_BE_RANKED = "${maxOptionsToBeRanked}";
+            public static final String RANK_IS_MIN_OPTIONS_TO_BE_RANKED_ENABLED = "${isMinOptionsToBeRankedEnabled}";
+            public static final String RANK_IS_MAX_OPTIONS_TO_BE_RANKED_ENABLED = "${isMaxOptionsToBeRankedEnabled}";
+            public static final String RANK_DISPLAY_MIN_OPTIONS_HINT = "${displayMinOptionsHint}";
+            public static final String RANK_DISPLAY_MAX_OPTIONS_HINT = "${displayMaxOptionsHint}";
+            public static final String RANK_MIN_RECIPIENTS_TO_BE_RANKED = "${minRecipientsToBeRanked}";
+            public static final String RANK_MAX_RECIPIENTS_TO_BE_RANKED = "${maxRecipientsToBeRanked}";
+            public static final String RANK_DISPLAY_MIN_RECIPIENTS_HINT = "${displayMinRecipientsHint}";
+            public static final String RANK_DISPLAY_MAX_RECIPIENTS_HINT = "${displayMaxRecipientsHint}";
+            public static final String RANK_IS_MIN_RECIPIENTS_TO_BE_RANKED_ENABLED = "${isMinRecipientsToBeRankedEnabled}";
+            public static final String RANK_IS_MAX_RECIPIENTS_TO_BE_RANKED_ENABLED = "${isMaxRecipientsToBeRankedEnabled}";
             public static final String RANK_PARAM_TO_RECIPIENT = "${Const.ParamsNames.FEEDBACK_QUESTION_RANKTORECIPIENTS}";
             public static final String RANK_PARAM_NUM_OPTION =
                     "${Const.ParamsNames.FEEDBACK_QUESTION_RANKNUMOPTION}";
             public static final String RANK_PARAM_IS_DUPLICATES_ALLOWED =
                     "${Const.ParamsNames.FEEDBACK_QUESTION_RANKISDUPLICATESALLOWED}";
+            public static final String RANK_PARAM_OPTION = "${Const.ParamsNames.FEEDBACK_QUESTION_RANKOPTION}";
+            public static final String RANK_PARAM_NUMBER_OF_CHOICE_CREATED =
+                    "${Const.ParamsNames.FEEDBACK_QUESTION_NUMBEROFCHOICECREATED}";
```

Comment:
May not be related, but are these two params still used?

Reply:
Yes these seem to be used. [FEEDBACK_QUESTION_RANKOPTION](https://github.com/TEAMMATES/teammates/search?utf8=%E2%9C%93&q=FEEDBACK_QUESTION_RANKOPTION&type=) [FEEDBACK_QUESTION_NUMBEROFCHOICECREATED](https://github.com/TEAMMATES/teammates/search?utf8=%E2%9C%93&q=FEEDBACK_QUESTION_NUMBEROFCHOICECREATED&type=)